lib/go/templates/template_utils.go

```diff
@@ -21,6 +21,8 @@ const (
 	placeholderDUCAddress             = "0xDUCADDRESS"
 	placeholderForwardingAddress      = "0xFORWARDINGADDRESS"
 	placeholderMetadataViewsAddress   = "0xMETADATAVIEWSADDRESS"
+	placeholderTopShotLockingAddress  = "0xTOPSHOTLOCKINGADDRESS"
+	placeholderFakeTopShotAddress     = "0xFAKETOPSHOTADDRESS"
```

Comment:
what does fakeTopshotAddress do?

Reply:
fake topshot address is used by the last test in `locking_test`: "Should not be able to lock a non-TopShot.NFT". It is used in testing to deploy a second copy of the TopShot address as a "fake or inauthentic" TopShot contract. I then mint moments from it and try to lock them. It exercises the check that the locking contract will only hold TopShot.NFTs.

Reply:
ha, nice test